mapComposer/README.md

```diff
@@ -0,0 +1,111 @@
+# Map Composer plugin
+
+## Introduction
+
+The "Map Composer" plugin allows to create cartographic documents (usually called "a map"). Here, such a document is made of:
+* a map context: the content displayed in the map window of OrbisGIS. A map context is structured as an .ows file, which fit with the [OGC OWS-Context](http://www.opengeospatial.org/standards/owc) standard,
+* additionnal elements such as title, scale, orientation, ...
```

Comment:
A custom version of OWS-C available at https://github.com/orbisgis/ogc-custom-jaxb/tree/master/src/main/resources/owc